charts/spring-service/templates/deployment.yaml

```diff
@@ -125,6 +125,16 @@ spec:
                 configMapKeyRef:
                   name: tracing
                   key: zipkin-base-url
+            - name: "MANAGEMENT_TRACING_ENABLED"
+              valueFrom:
+                configMapKeyRef:
+                  name: tracing
+                  key: enabled
+            - name: "MANAGEMENT_ZIPKIN_ENDPOINT"
+              valueFrom:
+                configMapKeyRef:
+                  name: tracing
+                  key: zipkin-endpoint
```

Comment:
this also needs to be created in the current configmap!
The value is different, it's not a baseurl anymore but the fullpath. It should have the value `...<currentbaseurl>/api/v2/spans`